### DISCUSSION TITLE
Ventura Install on 650 g1

### DISCUSSION THREAD

**Discussion by peekpt** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#169] [Q&A]
1. Does it need Monterey installed?
2. My monetary install mimics macbookpro12,1 which is not compatible with ventura. in case of an upgrade the platform serials will work?
3. Does this install generates random platform serials, or we need to change it later?
4. Is it safe to use it with iCloud?

**Comment by chris1111** [comment]
[Follow the guide](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/blob/master/macOS%20Ventura%20Support.md)
You have to create a USB install drive with the program

**Comment by peekpt** [comment]
I know the guide, what I am asking is does it need to have Monterey already installed or can we make a fresh install?
Also I noticed the screenshots are showing same serial, so I presume we need to generate new serial, right?

**Reply by chris1111** [reply]
You have to create a USB install drive with the program
There is no probleme here I made a complete guide from the 650 G1 for ventura, what you need more^

**Comment by peekpt** [comment]
My question is clear do I need to install it on top of Monterey or in a clean disk.
My other question how serials are handle? Because I noticed same serials in the screenshots of your post about g1

**Comment by chris1111** [comment]
Your question is useless because I explain on the Ventura Guide that you have to do a USB installation
For serial numbers you can use whatever you want

**Comment by peekpt** [comment]
So clean Install.
And we need to change serials later or will happen to hack some else's account.

**Comment by chris1111** [comment]
You can't hack someone's account by serial numbers it doesn't make any sense, the only thing that happens if laptops have the same numbers you'll get a pop up from Icloud to reset. It happened to me last week, I simply changed my SMBIOS then delete the Old device from my acount
